### PR TITLE
feat: add initial standalone badge

### PR DESCRIPTION
**Pull request by roth1002**
如同今天跟 @joshuatai 等人開會結論

會請Designer將 Badge display: absolute的使用情境列出

這個就先做 Standalone
螢幕快照 2020-02-07 上午11 04 24

### CODE REVIEW

File: packages/styled-docs/pages/badge.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Badge
+
+Badges are used to highlight an item's status for quick recognition.
+
+## Import
+
+```js
+import { Badge } from "@chakra-ui/core";
```
Comment (austinhung81) [inline packages/styled-docs/pages/badge.mdx]: Is this should be changed to import from "@trendmicro/react-styled-core/Badge"?

Comment (roth1002) [reply packages/styled-docs/pages/badge.mdx]: XDDD 忘了改了

**Comment by cheton** [comment]
> Pass the `variantColor` prop to customize the color of the Badge. `variantColor`
can be any **color key** that exists in the `theme.colors`.

@roth1002 
From what's mentioned above, is it able to pass `variantColor="green.20"` or [other colors](https://github.com/cheton/styled-ui/blob/dev/packages/styled-theme/src/base.js#L54-L189) to the Badge component?

**Comment by roth1002** [comment]
> > Pass the `variantColor` prop to customize the color of the Badge. `variantColor`
> > can be any **color key** that exists in the `theme.colors`.
> 
> @roth1002
> From what's mentioned above, is it able to pass `variantColor="green.20"` or [other colors](https://github.com/cheton/styled-ui/blob/dev/packages/styled-theme/src/base.js#L54-L189) to the Badge component?

I

**Comment by roth1002** [comment]
> > > Pass the `variantColor` prop to customize the color of the Badge. `variantColor`
> > > can be any **color key** that exists in the `theme.colors`.
> > 
> > 
> > @roth1002
> > From what's mentioned above, is it able to pass `variantColor="green.20"` or [other colors](https://github.com/cheton/styled-ui/blob/dev/packages/styled-theme/src/base.js#L54-L189) to the Badge component?
> 
> I

螢幕快照 2020-02-07 上午11 04 24

**Review by cheton — APPROVED** [review]

**Pull request merged**